unit/suites/box.lua

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env tarantool
-os = require('os')
+local os = require('os')
```

Comment:
The same warning is reported for `test/cluster-py/master.lua`.

Reply:
Fixed. Verified this way: `luacheck . --exclude-files test-run --read-globals box`.